client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -111,3 +113,20 @@ def test_get_access_token(self, client):
         token = endpoint.get_access_token()
 
         assert token is None
+
+    def test_get_deployed_model(self, client, experiment_run, model_for_deployment):
```

Comment:
Let's use the `created_endpoints` fixture to cleanup after the test ([example](https://github.com/VertaAI/modeldb/blob/412369f/client/verta/tests/test_endpoint/test_endpoint.py#L32-L36)), until I figure out an easier way to handle cleanups...

Reply:
fixed